mozilla/foundation.mozilla.org
# updated from push to unshift to have correct order

# Description



Link to sample test page: http://localhost:8000/privacynotincluded
Related PRs/issues: #9691


This PR updates the JS logic which creates the array used in the "More Categories" dropdown menu.

Instead of using `.push()`, we are now using `.unshift()`, as prepending the items adds them in the correct order.


# Steps to test

1. Visit the CMS and take a look at the PNI Product Category Snippet page. Note the order that they are in.
2. Visit the PNI home page, and take a look at the category navigation, they should be in the same order as the CMS, categories within the "more categories" dropdown included.
3. If you would like to test further, update the order of the categories in the CMS and revisit the navigation.
4. If everything is working as expected, testing is complete!

# Screenshots

Order in the CMS:
<img width="305" alt="Screenshot 2022-11-14 at 1 45 04 PM" src="https://user-images.githubusercontent.com/18314510/201777386-6e58fb04-ff3f-4b69-9362-48445ef5fc77.png">

Order on the front end:
<img width="842" alt="Screenshot 2022-11-14 at 1 59 56 PM" src="https://user-images.githubusercontent.com/18314510/201777392-bd939ff4-3507-4a6e-9551-975e857d8295.png">

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**
